test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -160,7 +160,16 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'telemetry.sendUsageTo (any)',
         'usageCollection.uiCounters.debug (boolean)',
         'usageCollection.uiCounters.enabled (boolean)',
+        'vis_type_gauge.readOnly (any)',
```

Comment:
Question: Are these `any` instead of `boolean` because they are optional?

Reply:
Would you mind adding an in-line comment to these new entries explaining that they are generally a `boolean`, but not required so they are `any`?

Reply:
comment added